docs/guidelines/content-guidelines/04-ui-elements.md

```diff
@@ -0,0 +1,110 @@
+
+- Keep your tooltips 1-2 sentences long. If you need to explain a concept in more details, create a separate document and link to it instead of providing the whole explanation in a tooltip.
+- Use sentence case inside a tooltip and follow punctuation rules.
+- Use tooltips to provide examples, for example:
+
+  ✅ The example of a domain name is `yourdomain.example.com`.
+
+- Omit punctuation in case of one-line label tooltips. For more information, see [Labels](#labels).
+
+
+## Messages
+
+There are many different types of messages appearing in every UI. The most common ones are pop-ups and regular messages that  users can find on your page. To avoid information pollution, make sure that all your messages are informative and useful to the users. Moreover:
+
+- Use punctuation in case of messages that are full sentences. In case of clauses (fragments of a sentence), omit punctuation.
+- Use title case for the title of your pop-up message. Keep it as simple as possible and omit punctuation. For more information, see [Labels](#labels).
+
+### Error messages
+
+Error message is a special type of message you must be really careful about. It informs about unsuccessful outcome of an action or conveys any other negative information, so you must choose your words wisely not to upset the user. Here are some tips that will help you design your error message:  
+
+- Describe what happened, why it happened, and what the user can do to fix the occurred issue. Avoid abstract messages that only inform that something went wrong. This could make the user feel frustrated not only about the fact that the error has occurred, but also for the fact they cannot do anything about it. However, do not describe the whole troubleshooting in an error message. Link to a separate troubleshooting guide instead.
+- Avoid technical terms and jargon. Every user should be able to understand the message, regardless their technical knowledge. Use language as plain as possible.  
+- Don't blame the user for making an error. Focus on the problem instead, for example:
+
+  ✅ Your password is incorrect. <br>
+  ⛔️ You have entered an incorrect password.
+
+- Avoid title case. It can give users a feeling they are being looking down on. (what about labels/titles? to clarify)
+
+
+## Placeholder texts
+
+A placeholder is a tricky UI element that in general should be avoided. Placeholders tend to disappear when the user clicks a given form field, so it requires them to use their memory, which increases memory load and hurts usability.
```

Comment:
I think in Busola placeholders are quite important. It contains a lot of forms to fill in.

Reply:
Rephrased